dusty/daemon.py

```diff
@@ -61,6 +66,7 @@ def _listen_on_socket(socket_path, suppress_warnings):
                     suppress_warnings |= payload['suppress_warnings']
                     logging.info('Received command. fn: {} args: {} kwargs: {}'.format(fn.__name__, args, kwargs))
                     try:
+                        _refresh_warnings()
```

Comment:
Well warnings never get changed after preflight, even if someone upgrades the version of one of boot2docker, nginx, etc.  which is confusing